src/custom/CatalogDetail/LeftPanel.tsx

```diff
@@ -22,6 +22,8 @@ interface LeftPanelProps {
   technologySVGSubpath: string;
   fontFamily?: string;
   showOpenPlaygroundButton?: boolean;
+  handleUnpublish: () => void;
```

Comment:
we can make it to optional because the show unpublished action button is optional.